cluster/charts/universal-crossplane/templates/crossplane/service.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.metrics.enabled }}
```

Comment:
@ulucinar the crossplane chart is copied verbatim from upstream in the current machinery. We would either need to switch from the direct copy, or (more likely) support a supplemental manifests directory for the chart that does not get overwritten.